Gemfile

```diff
@@ -5,7 +5,7 @@ gem "git", ">= 1.2.5"
 gem "nokogiri", ">= 1.5.10"
 gem "github_api"
 gem "highline", ">= 1.6.15"
-gem "bundler", ">= 1.0"
+gem "bundler", ">= 1.13"
```

Comment:
I think it should be a bit more general and made 
gem "bundler", ">= 1"
What are your thoughts on this?